src/domain/app/i18n/en.json

```
@@ -1438,6 +1442,7 @@
     "permission": "permission",
     "place": "place",
     "platform": "platform",
+    "priceGroup": "price, group",
```

```suggestion
    "priceGroup": "price group",
```
I suppose comma was not meant to be here?

---

Changed to `customer group` to unify the terms